packages/amazon-bedrock/package.json

```diff
@@ -0,0 +1,72 @@
+{
+  "name": "@ai-sdk/amazon-bedrock",
+  "version": "0.0.29",
+  "license": "Apache-2.0",
+  "sideEffects": false,
+  "main": "./dist/index.js",
+  "module": "./dist/index.mjs",
+  "types": "./dist/index.d.ts",
+  "files": [
+    "dist/**/*",
+    "internal/dist/**/*"
+  ],
+  "scripts": {
+    "build": "tsup",
+    "clean": "rm -rf dist && rm -rf internal/dist",
+    "dev": "tsup --watch",
+    "lint": "eslint \"./**/*.ts*\"",
+    "type-check": "tsc --noEmit",
+    "prettier-check": "prettier --check \"./**/*.ts*\"",
+    "test": "pnpm test:node && pnpm test:edge",
+    "test:edge": "vitest --config vitest.edge.config.js --run",
+    "test:node": "vitest --config vitest.node.config.js --run"
+  },
+  "exports": {
+    "./package.json": "./package.json",
+    ".": {
+      "types": "./dist/index.d.ts",
+      "import": "./dist/index.mjs",
+      "require": "./dist/index.js"
+    },
+    "./internal": {
+      "types": "./internal/dist/index.d.ts",
+      "import": "./internal/dist/index.mjs",
+      "module": "./internal/dist/index.mjs",
+      "require": "./internal/dist/index.js"
+    }
+  },
+  "dependencies": {
+    "@ai-sdk/provider": "0.0.10",
+    "@ai-sdk/provider-utils": "0.0.14",
+    "@aws-sdk/client-bedrock-runtime": "^3.598.0"
```

Comment:
preference is usually to call the api directly using fetch, unless it's not easily possible because of e.g. auth. does the bedrock runtime have any limitations (e.g. does it work on edge)?

Reply:
There is a rest API for these calls so I could use fetch but I would have to convert the auth and I also would have to define all the types the library provides explicitly.

Reply:
auth can make this fairly complicated and is the reason why i used a similar approach for the vertex provider. fine to leave as is.

btw, would you mind documenting how auth works? (e.g. in a separate file, or in something similar to https://github.com/vercel/ai/blob/main/content/providers/01-ai-sdk-providers/11-google-vertex.mdx

Reply:
Added documentation with auth and model capabilities etc.